plugins/ltac/g_rewrite.mlg

```diff
@@ -87,7 +87,8 @@ let pr_glob_strategy env sigma prc prlc _ (s : glob_strategy) =
 
 }
 
-ARGUMENT EXTEND rewstrategy
+ARGUMENT EXTEND rewstrategy0
```

Comment:
This code shows why this PR is nonsense: we now have a spurious generic argument registering a crapload of data that will never be used alone.